docs/blog/gazette-july-2020/index.md

```diff
@@ -0,0 +1,83 @@
+https://twitter.com/partycorgipod/status/1288087382504419335
+
+[Matías Hernández Arellano](https://matiashernandez.dev/) authored an Egghead course on creating a Gatsby source plugin. You can watch it in [English](https://egghead.io/playlists/creating-a-gatsby-source-plugin-3f01) or [in Spanish](https://egghead.io/playlists/creacion-de-un-plugin-de-gatsby-desde-cero-5c8b).
+
+Arshad is back with another Gatsby Theme for to his growing [ReflexJS](https://reflexjs.org/) collection. This new theme adds a video section to your site, and is optimized with SEO, OG and Twitter metatags. Reflex also comes with beautiful, [ready-to-use components](https://reflexjs.org/library/blocks) so you can customize the style and layout of your site quickly.
+
+![Reflex screenshot](./reflex.png)
+
+[Wojciech Kocjan](https://kocjan.net/)'s "Intro" theme crossed a major milestone in July with 1,000 downloads 🎉. If you need a well-designed and simple resume and portfolio site, [spin up Intro](https://github.com/wkocjan/gatsby-theme-intro).
+
+## 🪐 New from the Gatsby Ecosystem
+
+### Rewrite of GraphCMS’s source plugin
+
+We helped GraphCMS build their new source plugin ([using our new GraphQL toolkit](http://github.com/vladar/gatsby-graphql-toolkitgithub.com/vladar/gatsby-graphql-toolkit)). Head to the repo, and [give it a try](https://github.com/GraphCMS/gatsby-source-graphcms/).
+
+### Strapi's Gatsby Blog Starter
+
+[Strapi released a gorgeous blog starter](https://strapi.io/blog/strapi-starter-gatsby-blog-v2) that you can use with their headless CMS with minimal setup. [Watch the demo on YouTube](https://youtu.be/wsFuSebpv0I).
+
+### Ink v3
+
+Ink, a project that enbales you to build command-line apps using React [released version 3](https://vadimdemedes.com/posts/ink-3), a major upgrade. We use Ink for Gatsby's CLI, so you will soon features from this update flow into our project. If you ever wanted to build a CLI yourself, and you know React, [give Ink a try](https://github.com/vadimdemedes/ink).
+
+## 💫 Next from Gatsby
+
+### New alpha for Gatsby Recipes
+
+In the coming weeks look out for exciting updates to [Gatsby Recipes](/blog/2020-04-15-announcing-gatsby-recipes/), our new [infrastructure-as-code](docs/glossary/infrastructure-as-code/) project. You can follow along with our progress in our [public GitHub Project](https://github.com/gatsbyjs/gatsby/projects/20)
+
+### MDX Mini-Conference
+
+The maintainers of MDX are throwing their first ever conference! It’s free, it will be streamed remotely (because covid…), and you’ll hear from Gatsby engineers like [Josh Comeau](https://joshwcomeau.com/) and [Laurie Barth](https://laurieontech.com/)
```

Comment:
I wonder if we should include a link to the MDX conf landing page for folks that are interested?

```suggestion
The maintainers of MDX are throwing their [first ever conference](https://mdxjs.com/conf/)! It’s free, it will be streamed remotely (because covid…), and you’ll hear from Gatsby engineers like [Josh Comeau](https://joshwcomeau.com/) and [Laurie Barth](https://laurieontech.com/)
```